.gitignore

```diff
@@ -19,3 +19,5 @@
 /_site/
 /build/
 /out-tsc/
+
+/public/cim/
```

Comment:
More like a remark, it would be nicer to add the following line to the .gitignore
!/publi/cim/README.md

This way it's clear that the README.md should be there.